src/Skedify.test.js

```diff
@@ -333,6 +333,16 @@ describe('API', () => {
           )
         )
       ).toMatchSnapshot()
+      expect(
```

Comment:
I don't think this change is necessary because otherwise we have to add all resources and all includes and all combinations which becomes a bit messy. (I'll push a commit to remove this again).